### PR TITLE
Fix regression in b57f1d6

### PR DESCRIPTION
**Pull request by XhmikosR**
@sindresorhus: sorry for this... I should have had my coffee first :/

Please also release a new patch release.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,10 +18,7 @@ const loadPlugin = (plugin, ...args) => {
 	}
 };
 
-const exposePlugin = plugin =>
-	function (...args) {
-		return loadPlugin(plugin, args);
-	};
+const exposePlugin = plugin => (...args) => loadPlugin(plugin, args);
```
Comment (XhmikosR) [inline index.js]: Hmm this doesn't seem to work @sindresorhus either

Comment (sindresorhus) [reply index.js]: My change here is correct. See: https://github.com/sindresorhus/gulp-imagemin/commit/c7170ba2cd03f888e30e79dbbc52ee7983d99300

Comment (XhmikosR) [reply index.js]: I'm still getting wrong results with 5.0.2 in a repo of mine where I pass extra options

```js
imageoptim.svgo.js2svg = {
    indent: 2,
    pretty: true
  };
```

Then I still get the SVGs not beautified

**Comment by sindresorhus** [comment]
I would recommend adding a test for this to prevent mistakes like this in the future.

**Comment by XhmikosR** [comment]
Agreed. Just trying to fix the regression for the time being. This does work for me https://github.com/sindresorhus/gulp-imagemin/compare/fix

**Pull request merged**